gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowExecutionResource.java

```diff
@@ -63,6 +63,12 @@ public FlowExecution get(ComplexResourceKey<FlowStatusId, EmptyRecord> key) {
     return this.flowExecutionResourceHandler.getLatestFlowExecution(context, flowId, count, tag, executionStatus);
   }
 
+  @Finder("latestFlowGroupExecutions")
+  public List<FlowExecution> getLatestFlowGroupExecutions(@Context PagingContext context, @QueryParam("flowGroup") String flowGroup,
+      @Optional @QueryParam("countPerFlow") Integer countPerFlow, @Optional @QueryParam("tag") String tag) {
```

Comment:
Should this new finder also accept 'executionStatus' as another param ?

Reply:
I punted on that... and was unsure with all the layering where to put javadoc to that effect.  finding more on the `FlowExecutionResourceHandler` than here, I added this note in that location - 

```
   * NOTE: `executionStatus` param not provided yet, without justifying use case, due to complexity of interaction with `count`
   * and resulting efficiency concern of performing across many flows sharing the single named group.
```
so basically I was waiting for a use case.  I do note that the `executionStatus` on `latestFlowExecution` remains unexercised by our existing CLI client, so I definitely didn't want to pursue replicating something that itself may see little usage.

do note that if we decide to add this param here later, it should preserve b/w compatibility on this finder.

Reply:
Okay